static/app/components/replays/replayController.tsx

```diff
@@ -146,24 +146,27 @@ function ReplayOptionsMenu({speedOptions}: {speedOptions: number[]}) {
   );
 }
 
-function TimelineSizeBar({size, setSize}) {
+function TimelineSizeBar() {
+  const {timelineSize, setTimelineSize} = useReplayContext();
   return (
     <ButtonBar merged>
       <Button
         size="xs"
         title={t('Zoom out')}
         icon={<IconSubtract size="xs" />}
         borderless
-        onClick={() => setSize(Math.max(size - 50, 100))}
+        onClick={() => setTimelineSize(Math.max(timelineSize - 50, 100))}
         aria-label={t('Zoom out')}
+        disabled={timelineSize === 100 ? true : false}
```

Comment:
The left side of the ternary returns a boolean, no need for the `true` and `false` keywords :D

```suggestion
        disabled={timelineSize === 100}
```